frame/support/src/traits/tokens/currency/reservable.rs

```diff
@@ -81,3 +81,65 @@ pub trait ReservableCurrency<AccountId>: Currency<AccountId> {
 		status: BalanceStatus,
 	) -> Result<Self::Balance, DispatchError>;
 }
+
+/// An identifier for a reserve. Used for disambiguating different reserves so that
+/// they can be individually replaced or removed.
+pub type ReserveIdentifier = [u8; 8];
```

Comment:
would be better if this were an associated type since we do plan on altering this type in the future.